projects/fine_tuning/testing/config.yaml

```
@@ -161,6 +161,38 @@ ci_presets:
     tests.fine_tuning.test_settings.hyper_parameters.raw_lists: {target_modules: ["q_proj", "k_proj"]}
 
 
+  ibm_qlora_models:
+    extends: [hf_evaluation, dgx_small_footprint]
+    tests.fine_tuning.test_settings.model_name: null # for consistency. Set in 'test_extra_settings'
+    tests.fine_tuning.test_settings.dataset_name: alpaca_data.json
+    tests.fine_tuning.test_settings.dataset_replication: .5
+    tests.fine_tuning.test_settings.gpu: [8]
+    tests.fine_tuning.test_settings.hyper_parameters:
+      peft_method: "lora"
+      use_flash_attn: true
+      max_steps: -1
+      per_device_train_batch_size: [1] # this for now, until it works
+      max_seq_length: [512]
+      gradient_accumulation_steps: 4
+      warmup_ratio: 0.03
+      num_train_epochs: 1
+      gradient_checkpointing: true
+      packing: false
+      r: 4
+      lora_alpha: 16
+      # qlora arguments: https://github.com/foundation-model-stack/fms-hf-tuning?tab=readme-ov-file#fms-acceleration
+      torch_dtype: "float16"
+      fp16: true
+      log_level: "debug"
+    tests.fine_tuning.test_extra_settings:
+    - name: meta-llama3-8b-gptq
+      model_name:
+      - TechxGenus/Meta-Llama-3-8B-GPTQ
+      hyper_parameters:
+        raw_lists:
+          target_modules: ["all-linear"]
+          auto_gptq: ["triton_v2"]
```

```suggestion
    tests.fine_tuning.test_settings.hyper_parameters:
      peft_method: "lora"
      use_flash_attn: true
      max_steps: -1
      per_device_train_batch_size: 1 # this for now, until it works
      max_seq_length: [512, 1014]
      gradient_accumulation_steps: 4
      warmup_ratio: 0.03
      num_train_epochs: 1
      gradient_checkpointing: true
      packing: false
      r: 4
      lora_alpha: 16
      # qlora arguments: https://github.com/foundation-model-stack/fms-hf-tuning?tab=readme-ov-file#fms-acceleration
      torch_dtype: "float16"
      fp16: true
      log_level: "debug"
      raw_lists:
        auto_gptq: ["triton_v2"]
    tests.fine_tuning.test_extra_settings:
    - name: meta-llama3-8b-gptq
      model_name: TechxGenus/Meta-Llama-3-8B-GPTQ
      hyper_parameters.raw_lists:
          target_modules: ["all-linear"]
```

@albertoperdomo2 , can you try with this configuration and let me know if it works better? (there are 2 max_seq_length, on purpose)